requirements.txt

```diff
@@ -49,20 +50,24 @@ python-dateutil==2.8.0
 python-dotenv==0.10.3
 python-pptx==0.6.18
 pyunsplash==1.0.0b8
+PyYAML==5.1.2
```

Comment:
Why are there new requirements? :o

Reply:
let me give it a good cleaning. i removed and recreated requirements so an updated library might have brought in some dependencies.

Reply:
I think this is resolved now.

Reply:
Superb! Looks good to me! :D Thanks!